include/libdnf/rpm/solv_query.hpp

```diff
@@ -291,6 +298,11 @@ class SolvQuery {
     /// @replaces libdnf/sack/query.hpp:method:Query.size()
     std::size_t size() const noexcept;
 
+    // TODO(jmracek) return std::pair<bool, std::unique_ptr<libdnf::rpm::Nevra>>
+    /// @replaces libdnf/sack/query.hpp:method:std::pair<bool, std::unique_ptr<Nevra>> filterSubject(const char * subject, HyForm * forms, bool icase, bool with_nevra, bool with_provides, bool with_filenames);
+    std::pair<bool, libdnf::rpm::Nevra> subject_solution(const std::string & subject,
+        bool icase, bool with_nevra, bool with_provides, bool with_filenames, bool with_src, const std::vector<libdnf::rpm::Nevra::Form> & forms);
```

Comment:
`subject_solution` probably makes sense from a historical perspective as it refers to some objects that AFAIK no longer exist in the new codebase.
Shouldn't we rename it to `resolve_subject`?
From a high-level perspective - what's the difference between subject (as a string) and pkg_spec?
In such case, `resolve_pkg_spec` would make more sense.